.github/workflows/deploy.yml

```diff
@@ -0,0 +1,22 @@
+name: build
+
+on: push
+
+jobs:
+  test_and_deploy:
```

Comment:
github actions allow more modular CI & CD setup unlike travis , of course it is not a blocker for the release and I just wanted to share